geant4.sh

```diff
@@ -25,6 +25,10 @@ env:
 ---
 #!/bin/bash -e
 
+if [ ! $GEANT4_BUILD_MULTITHREADED ]; then
```

Comment:
Change those three lines to:

```bash
: {GEANT4_BUILD_MULTITHREADED:=OFF}
```

which does exactly the same. (Mind the `:` at the beginning of the line.)